patterns/README.md

```diff
@@ -1030,3 +1030,363 @@ A field MAY have a `missingValues` property that MUST be an `array` where each e
 ### Implementations
 
 None known.
+
+## Facilitate use of value labels (Stata, SAS and SPSS), categoricals (Python) and factors (R) in software that supports them
```

Comment:
This is a very long title. Cf. the other headings, I would shorten to:

```suggestion
## Table Schema: Enum labels and ordering
```

The software and the names used there are included in overview.